### PR TITLE
ajson, alacritty, alex: add French translation

### PR DESCRIPTION
**Pull request by Nico385412**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/alex.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# alex
```
Comment (marchersimon) [inline pages.fr/common/alex.md:1]: It might be worth mentioning that this tool only works in English

Comment (Nico385412) [reply pages.fr/common/alex.md:1]: why not! 
I added some words, and also fix some grammar errors ^^"

**Review by marchersimon — APPROVED** [review]
Thanks @Nico385412

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.
Since the requested reviewer hasn't responded yet, and it's a translation, not a trivial change. I am merging it into the main.

**Pull request merged**